docs/backend-api/resources/tracking/tracker/rules/rule.md

```diff
@@ -340,21 +341,23 @@ Updates rule and scheduled intervals.
+Presented parameters are common for all rules, but there are specific parameters that can be found in [rule types](./rule_types.md).
+
+| name            | description                                                                                                                                                                                                                                | type                                              |
+|:----------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------|
+| id              | ID of a rule. You can get IDs using the [rule/list](#list) call.                                                                                                                                                                           | int                                               |
+| name            | The name of created rule.                                                                                                                                                                                                                  | string                                            |
+| description     | Rule's description.                                                                                                                                                                                                                        | string                                            |
+| zone_ids        | List of zones to bind where the rule will work. Leave it empty if rule should work everywhere. Parameter `zone_ids` is not allowed for rule `offline` and required for `route` and `inoutzone` rule types (there can be exactly one item). | int array                                         |
+| trackers        | List of tracker IDs belong to user for which the rule will work.                                                                                                                                                                           | int array                                         |
+| type            | One of pre-defined types of rules. See [rule types](./rule_types.md).                                                                                                                                                                      | [enum](../../../../getting-started.md#data-types) |
+| primary_text    | Primary text of rule notification.                                                                                                                                                                                                         | string                                            |
```

Comment:
primary/secondary text now is not a common parameters

Reply:
Thank you, I have deleted them from rule/update API call too.